homeassistant/components/shelly/entity.py

```diff
@@ -634,7 +579,6 @@ async def async_added_to_hass(self) -> None:
 
         if last_state is not None:
             self.last_state = last_state.state
-            self.description.state_class = last_state.attributes.get(ATTR_STATE_CLASS)
```

Comment:
As discussed with @thecode, removed `state_class` as it would only apply to sensor platform, and it was a temporary workaround.